### PR TITLE
0 shown when appropriate asset selected [#4900]

### LINKED ISSUE #4900: 0  shown when appropriate asset selected (assets/collectibles are owned by selected ethereum address)
### Description

*Type*: Bug

*Summary*: Status wallet shows 0 collectibles after you select an asset even you have collectibles on your ethereum address. To make them show up I need to refresh assets manually using `pull-to-refresh` gesture. 

#### Actual behavior

Status wallet shows 0 collectibles after you select an asset even you have collectibles 

image

### Expected behavior

As a user, I want my collectibles to be shown right after I select appropriate asset (for example CryptoKittes). Apllies to any ERC20 token as well (for ex SNT)

### Reproduction

- Open Status
- Recover account with some collectibles (for ex CryptoKitties). 
- Buy some CryptoKitties if you don't own any
- Tap wallet -> Manage assets -> CryptoKitties
- Pay attention to ` Collectibles` section

Result: Collectibles section is displayed. CryptoKitties asset is shown. CryptoKitties amount is shown as 0.

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #4900

### CODE REVIEW

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/wallet/settings/views.cljs [hunk 1]
```diff
@@ -26,8 +26,10 @@
             visible-tokens [:wallet/visible-tokens-symbols]]
     [react/view (merge components.styles/flex {:background-color :white})
      [status-bar/status-bar {:type :modal-wallet}]
-     [toolbar/toolbar #_{} {:style wallet.styles/toolbar}
-      [toolbar/nav-text {:style               {:color :white}
+     [toolbar/toolbar {:style wallet.styles/toolbar}
+      [toolbar/nav-text {:handler             #(do (re-frame/dispatch [:update-wallet])
+                                                   (re-frame/dispatch [:navigate-back]))
```
Comment (jeluard) [inline src/status_im/ui/screens/wallet/settings/views.cljs:31]: Why `navigate-back` ?

Comment (flexsurfer) [reply src/status_im/ui/screens/wallet/settings/views.cljs:31]: what do you mean? it was navigate back by default, i added update-wallet

Comment (jeluard) [reply src/status_im/ui/screens/wallet/settings/views.cljs:31]: Ah of course!

**Review by jeluard — APPROVED** [review]

**Comment by Serhy** [comment]
@flexsurfer Looks good! Tokens and collectables (CK, STRK, EMONA) balance fetched and displayed right after navigate back from enabled respective asset to Wallet. No regression found (autotests addressed to be looked at separately - they are fine, and test_recover_account - expected as Feature Changed in the scope of https://github.com/status-im/status-react/commit/b34744133b87074566395693fbc67fd8e6d195aa).

Ready for the merge!

**Pull request merged**